pom.xml

```diff
@@ -23,7 +23,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>3.2.4</version>
+                <version>3.3.0-SNAPSHOT</version>
```

Comment:
Did you try to compile it using java 16+? The snapshot maven shade plugin is not required for java 11